apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.html.ts

```diff
@@ -582,7 +582,7 @@ export function InstructorAssessmentInstance({
                               <td>
                                 <button
                                   type="button"
-                                  class="badge color-${FINGERPRINT_COLORS[
+                                  class="btn badge color-${FINGERPRINT_COLORS[
```

Comment:
I'm not really sure about combining the `btn` and `badge` styling here. Would it be better to fully style these as buttons (probably `btn-sm` or `btn-xs`)? Or does that interfere with the colors?

Reply:
That makes sense and should not interfere with the colors. Updated to `btn-xs` in [3f9055a](https://github.com/PrairieLearn/PrairieLearn/pull/10853/commits/3f9055a3beb2f7f1a84941d6e7d020d63011604b).

Reply:
A quick search brought up this:
https://github.com/PrairieLearn/PrairieLearn/blob/38b83c3a1e5cb95451863ffaa6a531826eae745d/apps/prairielearn/src/pages/studentAssessmentInstance/studentAssessmentInstance.html.ts#L679
https://github.com/PrairieLearn/PrairieLearn/blob/38b83c3a1e5cb95451863ffaa6a531826eae745d/apps/prairielearn/elements/pl-integer-input/pl-integer-input.mustache#L76

So it's not unprecedented, but I see your concern. If this is changed to btn, though, we may want to revisit the cases above (though I believe the element case is being handled by #10701).

Reply:
I opened #10859 to fix the styling on the student assessment instance page. The styles in elements will indeed be fixed by the PR that you linked.